drivers/slipdev/slipdev.c

```diff
@@ -67,11 +67,95 @@ static inline void slipdev_unlock(void)
     }
 }
 
-static void _slip_rx_cb(void *arg, uint8_t byte)
+static inline void _slipdev_stdio_add_to_frame(slipdev_t * dev, uint8_t byte) {
+#if IS_USED(MODULE_SLIPDEV_STDIO)
```

Comment:
I think you can just do 
```suggestion
    if (!IS_USED(MODULE_SLIPDEV_STDIO)) {
        return;
    }
```

Reply:
I think both ways to do it are fine and valid? Personally I don't mind which way it is. Shall I change it to your proposal?